test-cases/gutenberg/reusable.md

```diff
@@ -0,0 +1,104 @@
+
+# Reusable Block - Test Cases
+
+## Preparation
+
+**For testing this block, it's required to create reusable blocks via the web version, here are the steps:**
+
+- Navigate to reusable blocks page (URL: `https://wordpress.com/types/wp_block/<YOUR_WPCOM_SITE_DOMAIN>`).
```

Comment:
I think that sometimes WPCOM will ask you to select the site anyway, like this:

![Screenshot 2021-05-27 at 20 41 06](https://user-images.githubusercontent.com/1032913/119872243-ec117c80-bf2b-11eb-8adf-97824b24ba7c.png)

So, what do you think adding a note about that in the instruction?

Reply:
Oh good point, I'll update the step then with:

- Navigate to reusable blocks page `https://wordpress.com/types/wp_block/`.
- Select the WP.com site for testing.

Reply:
I see this got addressed with https://github.com/wordpress-mobile/test-cases/pull/79/commits/d9411dd6cde52786887535b7104a17a6c3978d16#diff-6c976df25a3a675826c7dcc37f0922bd611004286dcba71e631192648fdd84c5R78-R79 👍 , thanks!